src/shared/lib/PtmUtils.ts

```diff
@@ -0,0 +1,73 @@
+import * as _ from 'lodash';
+
+import {PostTranslationalModification} from "shared/api/generated/GenomeNexusAPI";
+
+export const PTM_COLORS: {[type: string]: string} = {
+    "Phosphorylation": "#2DCF00",
+    "Acetylation": "#5B5BFF",
+    "Ubiquitination": "#B9264F",
+    "Methylation": "#EBD61D",
+    multiType: "#444444",
+    default: "#BA21E0",
+};
+
+export const PTM_PRIORITY: {[type: string]: number} = {
```

Comment:
@jjgao does this seem right? This is used to sort sub PTM tracks when expanded. 

Also we don't have the functionality to manually sort/hide sub tracks yet. Do you think we should try to add it for the initial version?